builtins/pcre/CMakeLists.txt

```diff
@@ -13,13 +13,19 @@ foreach(var PCRE_FOUND PCRE_VERSION PCRE_INCLUDE_DIR PCRE_PCRE_LIBRARY PCRE_LIBR
 endforeach()
 
 if(WIN32)
-  if(winrtdebug)
-    set(PCRE_POSTFIX $<$<CONFIG:Debug>:d>)
-    set(pcre_config_kind "Debug")
+  if(CMAKE_GENERATOR MATCHES Ninja)
+    if (CMAKE_BUILD_TYPE MATCHES Debug)
+      set(PCRE_POSTFIX d)
```

Comment:
is `d` required or arbitrary (if arbitrary consider using a longer form)

Reply:
It is the way pcre is naming its library in debug mode (adding the d suffix)